### PR TITLE
Adding gcloud CLI support to arm64 runner

### PR DESCRIPTION
**Pull request by freddan80**

### CODE REVIEW

**Comment by freddan80** [comment]
Needed by #15237 in order `gcloud cp` to work

**Review by ScottTodd — APPROVED** [review]

File: build_tools/github_actions/runner/gcp/image_setup.sh [hunk 1]
```diff
@@ -188,13 +188,25 @@ EOF
...
 
-  nice_curl \
-    https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-414.0.0-linux-x86_64.tar.gz \
-    --output gcloud.tar.gz
-  echo "${gcloud_checksum} *gcloud.tar.gz" | sha256sum --check --strict
-  tar -xf gcloud.tar.gz
-  rm gcloud.tar.gz
-  google-cloud-sdk/install.sh --quiet
+  GCLOUD_CLI_VERSION="414.0.0"
+  # Gcloud CLI versioned archive not available for Arm64 Linux
```
Comment (pzread) [inline build_tools/github_actions/runner/gcp/image_setup.sh]: I think https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-414.0.0-linux-arm.tar.gz might work. But feel free to keep the current solution.

**Review by pzread — APPROVED** [review]

File: build_tools/github_actions/runner/gcp/image_setup.sh (hunk 1 above)
Comment (freddan80) [reply build_tools/github_actions/runner/gcp/image_setup.sh]: Thx! I didn't see that one. I checked here: https://cloud.google.com/sdk/docs/downloads-versioned-archives, but it's not listed (I sent that feedback to the site...). I didn't find it via a Google search either.

Anyways, will fix.

**Comment by pzread** [comment]
Hi @freddan80 , thanks for fixing this. After you merge this PR, I will roll out the new image to runners, which should unblock #15237

**Comment by freddan80** [comment]
@pzread I can't merge, don't have write access :)

**Comment by freddan80** [comment]
Thx @pzread

**Pull request merged**